base/database/testing.go

```diff
@@ -12,7 +12,7 @@ func CheckAdvisoriesInDb(t *testing.T, advisories []string) []int {
 	err := Db.Where("name IN (?)", advisories).Find(&advisoriesObjs).Error
 	assert.Nil(t, err)
 	assert.Equal(t, len(advisoriesObjs), len(advisories))
-	var ids []int
+	var ids []int //nolint:prealloc
```

Comment:
You could pre-allocate here

Reply:
Yes, but this is only testing method, so I preferred simplicity over efficiency.